prep-css-flexbox/justify-content.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+#card-deck{
+display:flex;
+flex-basis:25%;
```

Comment:
`flex-basis` can only be applied to "flex items", that is the children of an element with `display: flex;` applied to it, so you will need a separate rule set with the selector for `.card-wrapper`.

Reply:
Ahh I see, updated work!

Align Items
![image](https://user-images.githubusercontent.com/97006645/149629581-e2b11daf-680d-4790-bcf3-70a3591875a0.png)
Justify Content
![image](https://user-images.githubusercontent.com/97006645/149629661-5bd4d22f-85cd-4835-a870-5d694724b27c.png)